### ISSUE TITLE
User roles

### ISSUE THREAD

**Issue by tejasmanohar** [sdelements/lets-chat#217]
Owner, Admin, Team Member, etc. User roles like Slack?

**Comment by JohnTheodore** [comment]
:+1:

**Comment by elhoucine** [comment]
:+1:

**Comment by elhoucine** [comment]
:

**Comment by afaqurk** [comment]
:+1:

**Comment by hsitaa** [comment]
Is its implementation being done. Or this issue is on hold for now. ?

**Comment by funkaoshi** [comment]
@hsitaa no one is working on this right now.

**Comment by hsitaa** [comment]
oh that means if the user forgets his password then its lost no recovery because no admin. So its all previous chats lost. and 
* it won't allow creation of new logins with same email id.
* No way to recover password or get the email id free to create new account.

Pretty lock down scenario. In case if want to use in live scenario. 
A big road block. coz password recovery is a most common issue at organizational Level.
Features required : 
*Password recovery
* Admin account or at least a way to use the email id's whose password has been forgotten.

Is there any way to deal with the issues currently ?

**Comment by funkaoshi** [comment]
Yes, we understand why this is useful. I'm just saying we have to prioritize the work we do. It's here in our issues because it's on our roadmap.

**Comment by mmeinert** [comment]
:soon: ?
I think this feature is important and will be usefull to all.

What's the current priority ?

Thank you devs

**Comment by tholu** [comment]
+1

**Comment by stefanffsx** [comment]
+1 Any news?

**Comment by Danaru87** [comment]
+1
